staff/staff_views_helper.py

```diff
@@ -358,3 +411,115 @@ def get_integer_days(class_offering):
+        )
+        / len(class_attendance)
+        if len(class_attendance) > 0
+        else 0
+    )
+    return round(average * 100, 2)
+
+
+def distribute_forms(request, posted_form, form):
+    groups = form.cleaned_data.get("recipient_groups")
+    for group in groups:
+        users = DjangoUser.objects.filter(groups__name=group.name)
+        for user in users:
+            create_form_distribution(posted_form, user)
+    classrooms = form.cleaned_data.get("recipient_classrooms")
+    for classroom in classrooms:
+        teacher = DjangoUser.objects.get(id=classroom.teacher_id)
+        create_form_distribution(posted_form, teacher)
+        teacher_assistant = DjangoUser.objects.get(id=classroom.teacher_assistant_id)
+        create_form_distribution(posted_form, teacher_assistant)
+        students = DjangoUser.objects.filter(
+            classroom_students__course=classroom.course
+        )
+        volunteers = DjangoUser.objects.filter(
+            classroom_volunteers__course=classroom.course
+        )
+        for student in students:
+            create_form_distribution(posted_form, student)
+        for volunteer in volunteers:
+            create_form_distribution(posted_form, volunteer)
+    messages.add_message(request, messages.SUCCESS, "Form Distributed Successfully")
+
+
+def create_form_distribution(posted_form, user):
+    dist = FormDistribution(
+        form=posted_form,
+        user=user,
+        submitted=False
+    )
+    dist.save()
+
+
+def get_outstanding_forms():
+    outstanding_form_dict = {}
+    for form in Form.objects.all():
+        distributions = FormDistribution.objects.filter(form_id=form.id)
+        outstanding_form_dict.update({form.name: distributions})
+    return outstanding_form_dict
+
+
+def email_form_notification(request, form, email_list):
+    subject = form.cleaned_data.get("subject")
+    msg_html = render_to_string(
+        "email_templates/post_form_email.html",
+        {
+            "subject": subject,
+            "message": form.cleaned_data.get("notification"),
+            "from": DjangoUser.objects.get(id=request.user.id),
+        },
+    )
+    text_content = "You are being notified about something"
+    recipient_list = [
+        "tyler.iams@gmail.com",
+        "iams.sophia@gmail.com",
+    ]  # Will replace with email_list
+    email = EmailMultiAlternatives(
+        subject, text_content, settings.EMAIL_HOST_USER, recipient_list
+    )
+    email.attach_alternative(msg_html, "text/html")
+    email.attach_file(str(Form.objects.get(name=request.POST.get("notify_about")).form))
+    email.send()
+    messages.add_message(request, messages.SUCCESS, "Recipients Successfully Emailed")
+
+
+def remove_dismissed_announcements(announcements, user):
+    res_announcements = []
+    for announcement in announcements:
+        ad = AnnouncementDistribution.objects.get(announcement_id=announcement.id, user_id=user.id)
+        if ad.dismissed is False:
```

Comment:
It's a bit more idiomatic to say `if not ad.dismissed:`, it's not typical to use `is` to test for booleans

Reply:
Fixed in d6173d2ff6c542d2319c5851cd57e41685b4b362